.travis.yml

```diff
@@ -68,8 +69,15 @@ jobs:
     - stage: deploy
       script:
         - travis_wait 40 ./.travis/.travis_nightly_build.sh
+        - travis_wait 60 ./.travis/.travis_te_nightly_build.sh
 
 env:
   global:
     - secure: R3NdobUAThkk6BBzXapek6PiuWHWa1ldnJBvXpk2TAi7Lua/Ctgg8EHCSve7+MK4SU3XFJSi1th32+VSy0K/ARfdVcxiVh6EJSL/Nw0LRQSvGk7VjO382SsWCFoJuZBBqjy9DfBgRLdoqEPfulIBC0XBxjJ+4xedRptkZo7+TAAvtB8Y32EA5ve+Z4cdJipLF1tviL5q2rVbkNXucYUYO2XIv5HOzF2Xy3Hw6lsNrHpbes2CLEkoMiLOQ30zfbXZT/5Xd6OhQNmoZJ9gKziT1opiN2AJMPv269K8FFJZTrDZX74/thq9wmeF4X4f3wPA7EbTupooj0i9LPzb8Xev1QKAsxBtHyRm9NGYOXM06c2yKiGsDtT4HZYQGk4m8/89moe3oGnEwvw/JLu5RvLIZzFE2cBhHrcxRAJgHLIr80+kebNg/YstNlDL+rmP0b9NRrGIVw5OO2BT27ZBgsMFmjnujQtCUJiJ5zqyMxygaYmgZha6xcbhFUPyf4DJ3T46etyFovg2EJnhM7HL9ECZHloQZFhEKs88sZXqvt6HKsA77xyL7ONMrwcPtdDVbFUkN/B1R7LLOf6Wq5vK/Y2EB18oXhKF0ngdahT8HKSMTXjuYf5MPybxhf4fdY6stFmIVkd9b633iZDMu+Aj8E1c8BApyDikCmYK2BI3grH49E0=
     - secure: Hj71ADg7ZgQpn8PUbEjaz1RrjkPfPCpGksp+WDjE6jf+oDvl55ILQhPgNcxIy81UcxS3Gmx6UWteVGkNV4Fi59TJIQ8KPbQ9RU2FFVXcxHH2TmkOvdNezwkJIFUw0TNMCbRAXgDj7ov75BM6lLi271GfqPVg9pB504Bc43WohnV6lDo/y93Z/xWbuFk5uJ+cogAJme3pvYHxBNlNzwIYnNAAqrEb+YKsRuux3qWLTlm1guT78g3C9wl/WCelTo1A3hGd12FqFWmowcoQlAyhbnFJ7u7qwfSNTSGCFLYbrUGooXqTZg2B+2QZORT2b5b0P9SXgB2a/UJxXmxkUzbmPGdUJipmQCEemlxwkPCqMW0YSscTDHU5WjSxqy4Cvwa8y+DEpLu06WxAPwjBje53025PvLjsMEfNJ0wsQbm9jkrH0alzICemXyyV5uGBCmuiMv5WqZ70WEpy7xlPlk5aFTwWQRpYJ5mhQCnHCgQfiX1/reEo6Vi5DzsW6A0Qtr5UERwMpQwYd47ErbobyuJY4Bcfz/TXqfaCY2+ER9kWil87dBTDpH5K8X68FHxIwNx2OxGVwlc3JhpMXJ2uFDDeXvsPlQA2kQWgZnC03qXSZX5YmTOjHoNREGtgEsgDZS1vM5M3KZkOG0+76MWL5p/zgHfVjzS3T15W4PVWlki/WZ0=
+    - secure: VfBYdV1EKmIBQlJj6yYUmucDU33HOm2dZLGcWC91QFVQx3FCOFzhSwTcby/gFM/9q/OzudKIRyTLV3iCtM1j1dEQA+oHLq6T1uV8DsOXnY908oShK1eil53aa3Ki4tYyMhR8WiXqVCoXglSB/Eakl6X2FnBfj0au+jvYLgPJE9IgY8hOvmp9ZdX5yFCbgcU2lmmJb7TZSnarrdtDBPuMG4dmRoqcUMxslvIZskOs38NVfkR4fj3gPGt2yZcauEKrqiiBF1nLGLfkigrifdHTW9fkIQtQmLwKrZx7+Mxz0GHAIrkGfoauQKfJy6fdFjyumtxWPxHKG1LlkmjrXEOustCefRy9S2bMkemgbCFT4jXsgHBCfmmV4BFADwvxcWSI6U9iSqeHfQMRgyvrJ36PaVi66UjVs+nPKPjMeHNcO32edB71/BKINEOgd38J1f4qTaUfnNexAf/lAT+rxl9PMfAuttDM5QGwdDfQa/6+hRnGJaPyHiAALFqLpgabJ3oRUsQA7D8DHb+Rk8vstsVEeuB0h/WrqepzKyPz38FqxbODAsLuhPv1hunecT/u7Vj2PZnLSoBF85cklEfYnqofu3eFJr2GyoY0Wa70PRw+lpGsrbBkiQoGkpehv7WEECUB3RZEighTHcBauAyR4dg8hmt29EXFOUXHrTrRaFZszp8=
```

Comment:
Where do those encrypted settings like `NPM_EMAIL` get used?

Reply:
`npm-login-noninteractive` is using three env variables, NPM_USER, NPM_EMAIL and NPM_TOKEN.